python/tidy/servo_tidy/tidy.py

```diff
@@ -483,6 +483,7 @@ def check_rust(file_name, lines):
             (r"\{[A-Za-z0-9_]+\};", "use statement contains braces for single import",
                 lambda match, line: line.startswith('use ')),
             (r"^\s*else {", "else braces should be on the same line", no_filter),
+            (r"(fn )?[A-Za-z0-9_<>]+\( [A-Za-z0-9]", "extra space after (", no_filter),
```

Comment:
What happens if we just match `( ` without any of the surrounding characters? Do we get false positives?